pillar/dagster/init.sls

```diff
@@ -0,0 +1,50 @@
+# -*- mode: yaml -*-
+{% set environment = salt.grains.get('environment') %}
+
+dagster:
+  home: /opt/ol_data_pipelines
+  dagit:
+    path: /opt/ol_data_pipelines/bin
+    flags: -w etc/workspace.yaml
+  config:
+    instance:
+      scheduler:
+        module: dagster_cron.cron_scheduler
+        class: SystemCronScheduler
+      compute_logs:
+        module: dagster_aws.s3.compute_log_manager
+        class: S3ComputeLogManager
+        config:
+          bucket: dagster-{{ environment }}/compute-logs/
+      run_storage:
+        module: dagster_postgres.run_storage
+        class: PostgresRunStorage
+        config:
+          postgres_db:
+            username: __vault__:cache:postgres-{{ environment }}/creds/app
+            password: __vault__:cache:postgres-{{ environment }}/creds/app
```

Comment:
Shouldn't the `app` be defined here?

Reply:
So, for the work I did on the VaultDatabase component in Pulumi I decided to just create a generic `app` role rather than creating a role defined based on the name of the app, which is what I'm leaning on here. Happy to reconsider that pattern if you or @markbreedlove has a preference.